### PR TITLE
Add FinancierDisplayWeb Light/Semibold support.

### PR DESCRIPTION
**Pull request by notlee**
Adds:

- MetricWeb Black
- FinancierDisplayWeb Light
- FinancierDisplayWeb Semibold
- FinancierDisplayWeb Black
- FinancierTextWeb Medium
- FinancierTextWeb Bold
- FinancierTextWeb Black

For backward compatibility MetricWeb and FinancierDisplayWeb variants are
output when no options are specified for the master brand. All MetricWeb are
output when no options are specified for the internal brand. In keeping with
this the new weights for these fonts will be output when no options are
specified:
```scss
@import 'o-fonts/main';
@include oFonts();
```

However there is no change when requesting the recommended font set for
master/internal products:
```scss
@import 'o-fonts/main';
@include oFonts($opts: ('recommended': true));
```

The new FinancierTextWeb fonts, added for specialist titles, will not be
output unless specifically asked for, e.g:
```scss
@import 'o-fonts/main';
@include oFonts($opts: (
	'metric': (
		(weight: 'black', style: normal)
	),
	'financier-text': (
                (weight: 'medium', style: normal),
                (weight: 'bold', style: normal),
                (weight: 'black', style: normal)
	)
));
```

Related specialist components PR: Financial-Times/specialist-components#275

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-fonts bundle size difference from 4.4.2 to a8eeae5e3a86ff0a7efbfa4c96aa5bc40773f607
css, master: 0.69kb increase (0.02kb/gzip)
An insignificant difference was also found for: css (internal), css (whitelabel)

**Review by JakeChampion — APPROVED** [review]

**Comment by notlee** [comment]
I'm going to hold off on releasing this until these extra fonts are included also:
https://github.com/Financial-Times/o-fonts-assets/pull/91

**Comment by origamiserviceuser** [comment]
o-fonts bundle size difference from 4.4.2 to 10cd33f8d3c5bdb2d610921479198e5fff2dfd79
css, master: 1.34kb increase (0.06kb/gzip)
css, internal: 0.31kb increase (0.02kb/gzip)
An insignificant difference was also found for: css (whitelabel)

**Comment by notlee** [comment]
I've updated this PR to include the extra fonts. It's a little more complicated now as no FinancierText font faces were output previously

File: README.md [hunk 1]
```diff
@@ -68,17 +68,17 @@ A selection wider than [recommended fonts](#recommended-fonts) are included by d
...
-| Weight   | FinancierDisplayWeb | MetricWeb |
-|----------|:-------------------:|:---------:|
-| thin     |                     |    ✓      |
-| light    |           i         |    ✓ i    |
-| regular  |         ✓ i         |    ✓ i    |
-| medium   |         ✓ i         |    ✓      |
-| semibold |           i         |    ✓      |
-| bold     |         ✓           |    ✓ i    |
-| black    |                     |           |
+Any of the below fonts may be included with `o-fonts` using [SASS](#sass), this is usual for brands such as specialist titles who may use additional fonts. Build Service users are limited to [fonts included by default](#fonts-included-by-default).
```
Comment (JakeChampion) [inline README.md:71]: @notlee, I can't tell if this is meant to say "usual" or "usually". I think both read well but wanted to check if you meant to write "usually" instead?
>Any of the below fonts may be included with `o-fonts` using [SASS](#sass), this is usual for brands such as specialist titles who may use additional fonts.

Comment (notlee) [reply README.md:71]: It's supposed to say "useful'? 🙈

Comment (notlee) [reply README.md:71]:
```suggestion
Any of the below fonts may be included with `o-fonts` using [SASS](#sass), this is useful for brands such as specialist titles who may use additional fonts. Build Service users are limited to [fonts included by default](#fonts-included-by-default).
```

**Comment by origamiserviceuser** [comment]
o-fonts bundle size difference from 4.4.2 to ba778b69569c103bf414352c03c344df471b04bc
css, master: 1.34kb increase (0.06kb/gzip)
css, internal: 0.31kb increase (0.02kb/gzip)
An insignificant difference was also found for: css (whitelabel)

**Comment by JakeChampion** [comment]
@notlee What is a black weight? Is that a term that came from the design team?

**Comment by notlee** [comment]
@JakeChampion the foundry / OpenType https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#common_weight_name_mapping

**Review by JakeChampion — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
o-fonts bundle size difference from 4.4.3 to 0acd42ffb1dddd61ac320ac2a04d4ac831ca1f99
No bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-fonts bundle size difference from 4.4.3 to f4987915286be4a714393a227ce17d331a95e117
No bundle size differences found.

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v4.5.0 :tada:

**Pull request merged**